scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id.py

```diff
@@ -66,6 +66,13 @@
     '010': 'country/USA'
 }
 
+_US_REPLACE_GEO_CODE_MAP = {
```

Comment:
Can we name this variable `_US_GEO_CODE_UPDATE_MAP`?

Reply:
sure